### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**
The Discordeno Documentation is a little outdated right now. Here I have updated the Documentation based on the latest Boilerplate features.
Also, some more changes were made/corrected:
- Formatting errors
- Deprecated imports
- Links to other parts of the Documentation
- Order in Step By Step guide

Some tiny things were changed too.

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes
(no code changes)

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
- [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: docs/src/advanced/arguments.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Discordeno comes with the most useful command arguments already built for you. T
 - `member` When you want a member object and the user can provide a member id, @mention or their username/nickname.
 - `number` When you want a number.
 - `role` When you want a role object. The user can provide the role id, mention or role name.
+- `snowflake` When you just want a snowflake.
```
Comment (Skillz4Killz) [inline docs/src/advanced/arguments.md:19]: You should maybe explain what a snowflake is 🤔  not everyone may know thats a unique generated id for users, channels, roles, guilds emojis etc

File: docs/src/advanced/arguments.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ Discordeno comes with the most useful command arguments already built for you. T
...
+- `..snowflakes` When you want to check if the mentioned ID is a valid snowflake.
```
Comment (Skillz4Killz) [inline docs/src/advanced/arguments.md:20]:
```suggestion
- `...snowflakes` When you want to check if the mentioned ID is a valid snowflake.
```

Comment (Skillz4Killz) [reply docs/src/advanced/arguments.md:20]: I would also clarify that this lets you collect all the snowlakes not just one and that this should only be used as the last argument.

File: docs/src/stepbystep/hostingbot.md [hunk 3]
```diff
@@ -64,5 +64,5 @@ deno install --allow-read --allow-run --allow-write --allow-net -f -q --unstable
 Once, that is done you can simply run the bot.
 
 ```shell
-denon run --allow-read --allow-net mod.ts
+denon run --allow-net --allow-read --no-check --config tsconfig.json mod.ts
```
Comment (Skillz4Killz) [inline docs/src/stepbystep/hostingbot.md:67]:
```suggestion
denon run --allow-net --allow-read --no-check mod.ts
```

File: docs/src/advanced/arguments.md (hunk 2 above)
Comment (ayntee) [reply docs/src/advanced/arguments.md]: That is irrelevant to the PR.

**Review by ayntee — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**